tests/scheduler/test_scheduler.py

```diff
@@ -0,0 +1,62 @@
+import time
+from datetime import datetime
+
+from dbos_transact import DBOS
+
+
+def test_scheduled_workflow(dbos: DBOS) -> None:
+    wf_counter: int = 0
+
+    @dbos.scheduled("* * * * * *")
```

Comment:
This runs every second, maybe also add a scheduled workflow that runs every 30min to test DBOS can be properly destructed.